applications/ConstitutiveLawsApplication/custom_constitutive/small_strain_isotropic_damage_3d.cpp

```diff
@@ -54,11 +54,12 @@ SmallStrainIsotropicDamage3D::~SmallStrainIsotropicDamage3D()
 bool SmallStrainIsotropicDamage3D::Has(const Variable<double>& rThisVariable)
 {
     if(rThisVariable == STRAIN_ENERGY){
```

Comment:
Replace with else if, or use swicth (with more than 4 cases it is mor eefficient)

Reply:
Done